examples/android/src/main/java/com/dropbox/core/examples/android/UserActivity.java

```diff
@@ -32,7 +32,11 @@ protected void onCreate(Bundle savedInstanceState) {
         loginButton.setOnClickListener(new View.OnClickListener() {
             @Override
             public void onClick(View v) {
-                Auth.startOAuth2Authentication(UserActivity.this, getString(R.string.app_key));
+                if (DropboxActivity.USE_SLT) {
```

Comment:
Can this be moved down into `DropboxActivity` so that less classes needed to know about `USE_SLT`?  Maybe a `starAuthentication` method on the base that this calls on click and it decides which `Auth` method to call?